### PR TITLE
ci(upgrade-test): bypass zoe-full-upgrade with hardcoded bundleIDs

### PR DESCRIPTION
**Pull request by michaelfig**
## Description

This temporarily turns off the failing `zoe-full-upgrade` CI until we can remove its dependence on hardcoded bundle hashes.

To reenable, use `grep -r FIXME agoric-sdk/packages/deployment/upgrade-test` as a guide.

cc: @Chris-Hibbert, @dckc, @turadg 

### Security Considerations

n/a

### Scaling Considerations

n/a

### Documentation Considerations

n/a

### Testing Considerations

Needs to be reenabled once the tests are fixed.

### Upgrade Considerations

n/a

### CODE REVIEW

File: packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh
Comment (mhofman) [inline packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh]: Can we move this at least after the `env_setup.sh` call below. I want to add to this upgrade test in an upcoming PR

**Review by mhofman — CHANGES_REQUESTED** [review]

**Comment by mhofman** [comment]
Also I believe the `test.sh` will now fail since the incarnation will have stayed at 0.

File: packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh
Comment (michaelfig) [reply packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh]: Sure, will do!

**Review by mhofman — APPROVED** [review]
Please wait until the docker upgrade test completes to hit the merge button :)

**Comment by michaelfig** [comment]
> Please wait until the docker upgrade test completes to hit the merge button :)

Latest problem at https://github.com/Agoric/agoric-sdk/actions/runs/5708296211/job/15465829100?pr=8116#step:4:31946
```
Step 64/68 : COPY --from=agoric-upgrade-10 /root/.agoric /root/.agoric
ApplyLayer exit status 1 stdout:  stderr: write /root/.agoric/data/application.db/000051.ldb: no space left on device
```

**Comment by michaelfig** [comment]
> Latest problem at https://github.com/Agoric/agoric-sdk/actions/runs/5708296211/job/15465829100?pr=8116#step:4:31946
> 
> ```
> Step 64/68 : COPY --from=agoric-upgrade-10 /root/.agoric /root/.agoric
> ApplyLayer exit status 1 stdout:  stderr: write /root/.agoric/data/application.db/000051.ldb: no space left on device
> ```

Solved via actions/virtual-environments#2840

**Pull request merged**
